### PR TITLE
Separated modded cars sql file

### PR DESCRIPTION
**Pull request by Xrez31**

### CODE REVIEW

**Comment by Xrez31** [comment]
Forgot to remove modded cars po from the other sql file hehe. Pushed updated sql file po.

File: new_vehicles.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
+INSERT INTO `vehicle_categories` (name, label) VALUES
+    ('high', 'High End'),
+	('mid', 'Mid End'),
+	('low', 'Low End'),
```
Comment (alvinveroy) [inline new_vehicles.sql]: remove the comma at the end of the SQL statement.

**Comment by Xrez31** [comment]
Removed comma at line 4

**Review by alvinveroy — CHANGES_REQUESTED** [review]
remove comma at the end of each SQL statement. @ line 4

**Pull request merged**